platforms/core-execution/build-cache-local/build.gradle.kts

```diff
@@ -0,0 +1,38 @@
+plugins {
+    id("gradlebuild.distribution.api-java")
+    id("gradlebuild.jmh")
+}
+
+description = "Local build cache implementation"
+
+dependencies {
+    api(project(":build-cache"))
+    api(libs.inject)
+
+    api(project(":base-annotations"))
+    api(project(":base-services"))
+    api(project(":build-cache-spi"))
+    api(project(":core-api"))
+    api(project(":files"))
+    api(project(":file-temp"))
+    api(project(":functional"))
+    api(project(":hashing"))
+    api(project(":persistent-cache"))
+    api(project(":resources"))
+
+    implementation(libs.guava)
+    implementation(libs.h2Database) {
+        because("Used in BuildCacheNG")
+    }
+    implementation(libs.hikariCP) {
+        because("Used in BuildCacheNG")
+    }
```

Comment:
Are Build cache NG dependencies still needed?

Reply:
They are included in the code right now (we have `H2BuildCacheService` in there). The dependency checker plugin would have flagged these otherwise.

A followup PR should be about publishing `:build-cache-local`. As part of that work we should make it depend on as little as possible. In that PR we should perhaps move out the BCNG stuff to yet another (not-published) subproject.